docs/remote-authentication/remote-authentication.md

```diff
@@ -52,6 +55,14 @@ Finally, if the ASP.NET Core app didn't previously include authentication middle
 app.UseAuthentication();
 ```
 
+## Securing the remote app connection
```

Comment:
This is duplicated below it seems. Can you structure the docs so that there's a common place for that?

Reply:
Yea, I wasn't sure the best place to put this in the docs since we don't have a page just for remote app connection (that isn't session- or auth- specific). I wasn't sure if it was worth adding one just for this, but I can make one. May come in handy later anyhow.

Reply:
Yeah, makes sense to now have it as a separate page since we've combined the concepts. Maybe on about "Remote App Setup" or something

Reply:
There's the usage-guidelines docs that had some of this as well. Maybe use that as a central point?